crates/polars-core/src/datatypes/dtype.rs

```diff
@@ -590,6 +590,8 @@ impl DataType {
             UInt64 => Scalar::from(u64::MAX),
             Float32 => Scalar::from(f32::INFINITY),
             Float64 => Scalar::from(f64::INFINITY),
+            #[cfg(feature = "dtype-time")]
+            Time => Scalar::new(Time, AnyValue::Time(86_399_999_999_999)),
```

Comment:
I think it might be clearer to use `NS_IN_DAY - 1` here instead of the literal value.